### PR TITLE
 EZP-28185: Doc for "Database update SQL missing from Kernel 6.12" 

### PR DESCRIPTION
**Pull request by mere-teresa**
Documentation for the PR https://github.com/ezsystems/ezpublish-kernel/pull/2143

Should be merged in 1.12 and master

### CODE REVIEW

File: docs/releases/updating_ez_platform.md [hunk 1]
```diff
@@ -212,29 +212,36 @@ index 49a17a9..80c4cd7 100644
...
-    ```
+    ​```
 
 **Increased password hash length**
 
 !!! caution "V1.12"
 
     v1.12.0 improves password security by introducing support for PHP's `PASSWORD_BCRYPT` and `PASSWORD_DEFAULT` hashing algorithms. By default `PASSWORD_DEFAULT` is used. This currently uses bcrypt, but this may change in the future as PHP adds support for new and stronger algorithms.
+    Apply the following database update script to change the schema and enable the storage of longer passwords:
+    Note that the script is available for PostGreSQL as well.
```
Comment (DominikaK) [inline docs/releases/updating_ez_platform.md]: `PostgreSQL` with small "g"
Also, right now this sounds like the same script also works for Postgres, so maybe:
`Note that there is a script available for PostgreSQL as well.`

**Comment by mere-teresa** [comment]
Conflicts and non-visible chars to fix.

**Comment by andrerom** [comment]
Seems it now contains all commits from master not in 1.12.

To fix:
```
git rebase -i 1.12
# delete all lines not relevant for this PR, in vim it is by pressing `d` when marker is on those lines and end with `:wq`
git push origin EZP-28185-database-update-sql-missing -f

**Comment by mere-teresa** [comment]
Okay, I made more mistakes with it, now.
Say, I am learning many things! (thank you @andrerom )

**Comment by mere-teresa** [comment]
See #93

**Pull request closed without merging**